client/my-sites/backup/index.js

```diff
@@ -36,7 +36,7 @@ export default function () {
 			backupActivity,
 			wrapInSiteOffsetProvider,
 			showUpsellIfNoBackup,
-			wpcomUpsellController( WPCOMUpsellPage ),
+			wpcomUpsellController( WPCOMUpsellPage, WPCOMBusinessAT ),
```

Comment:
@cbauerman commented something about the Activity Log not requiring the Upsell pages